admin/services/group.service.js

```diff
@@ -41,14 +41,14 @@ groupService.getGroupsAsArray = async function (schoolId) {
 /**
  * Get pupils filtered by schoolId and groupId.
  * @param schoolId required.  the school context
- * @param groupIdToExclude optionally exclude a single group from the returned set
+ * @param groupIdToInclude optionally exclude a single group from the returned set
```

Comment:
Param name does not match description: include / exclude

Reply:
fixed